deploy/crds/workspaces.ecd.eclipse.org_devworkspaces_crd.yaml

```diff
@@ -547,6 +547,26 @@ spec:
                         type: object
                     type: object
                   type: array
+                events:
+                  description: Bindings of commands to events
+                  properties:
+                    postStart:
+                      items:
+                        type: string
```

Comment:
What about aliasing `command.name` to e.g. `type CommandName string`, and then using it here? Maybe it's too complicated for little gain.

Reply:
These CRDs are generated by operator-sdk tools. So I'm not sure it is possible here. Can we open a distinct issue for this ?